Added info from support article about flex (#716)
hashicorp/terraform-docs-common: website/docs/cloud-docs/overview/activate-flex.mdx

There are several prerequisites for activating HashiCorp Flex:

> 1. You must email the HCP organization ID with the HashiCorp sales team. 

Reviewer: Hi @rkoron007 this pre req can be substituted with. “Create an HCP org.” Instructions. We no longer need to know the org id they will be using.

- The HCP admin offers HCP Terraform as a multi-tenant managed service. They manage the billing relationship with HashiCorp Cloud Platform and services customers of HCP Terraform.  The HCP admin should be invited to the HCP Terraform customer organizations to activate Flex.
- The HCP admin manages the billing relationship with HashiCorp Cloud Platform and owns all the HCP Terraform organizations. They can use either approach and invite HCP admins to HCP Terraform organizations or make HCP Terraform organization owners into HCP admins.

## Step 3: Activate your Flex plan

Your sales account manager will email your organization's tech contact a code to activate your HCP Flex plan a day or two before your plan begins. 

> You can self-activate your Flex plan by completing the following steps:

Reviewer 2: ```suggestion
Complete the following steps to activate your Flex plan:
```
Simpler?

1. Copy the HCP Flex activation code from your email.

> 1. Click the link in your activation email to access your HCP organization or log in to your HCP portal. Then, navigate to **Billing > Payment Methods** and find the **Contract** section.

Reviewer 2: ```suggestion
1. Click the link in your activation email to access your HCP organization or log into your HCP portal. 
1. Navigate to **Billing > Payment Methods** and find the **Contract** section.
```
